debian/README.Debian

```diff
@@ -5,7 +5,26 @@ This is a binary-only package produced by the wasm waterfall which
 is hosted at https://wasm-stat.us/.  The code that runs the build
 is hosted at https://github.com/WebAssembly/waterfall.
 
+This package includes the following components:
+
+ - upstream llvm including wasm backend (/opt/wasm/llvm)
+ - fastcmp llvm (/opt/wasm/fastcomp)
+ - emscripten (/opt/wasm/emscripten)
+ - binaryian (/opt/wasm/bin/)
```

Comment:
(Also binaryen has some components in `wasm-install/src/js` if we copy those to `/opt/wasm` too?)